src/components/results/securityanalysis/security-analysis-result-utils.ts

```diff
@@ -594,10 +594,26 @@ export const SECURITY_ANALYSIS_RESULT_INVALIDATIONS = [
     'securityAnalysisResult',
 ];
 
-export const FROM_COLUMN_TO_FIELD: Record<string, string> = {
-    subjectId: 'subjectId',
-    contingencyId: 'contingencyId',
-    status: 'status',
+export enum RESULT_TYPE {
```

Comment:
You didn't change it, put it back where it was ?

Reply:
well, since it's javascript, enum declarations need to be done before enum uses
it's either this, or I need to move "getColumnToFieldMapping" below
It doesn't matter to me though

Reply:
By setting getColumnToFieldMapping below the previous RESULT_TYPE, we can keep the same position and the original git history

Reply:
fixed